src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -57993,18 +57993,18 @@ static void G(C? x2, Unknown y2)
 }";
             var comp = CreateCompilation(new[] { source }, options: WithNullableEnable());
             comp.VerifyDiagnostics(
-                // (10,26): error CS0246: The type or namespace name 'Unknown' could not be found (are you missing a using directive or an assembly reference?)
+                // 0.cs(10,26): error CS0246: The type or namespace name 'Unknown' could not be found (are you missing a using directive or an assembly reference?)
                 //     static void G(C? x2, Unknown y2)
                 Diagnostic(ErrorCode.ERR_SingleTypeNameNotFound, "Unknown").WithArguments("Unknown").WithLocation(10, 26),
-                // (3,25): error CS0246: The type or namespace name 'Unknown' could not be found (are you missing a using directive or an assembly reference?)
+                // 0.cs(3,25): error CS0246: The type or namespace name 'Unknown' could not be found (are you missing a using directive or an assembly reference?)
                 //     static void F(C x1, Unknown? y1)
                 Diagnostic(ErrorCode.ERR_SingleTypeNameNotFound, "Unknown").WithArguments("Unknown").WithLocation(3, 25),
-                // (5,18): error CS0826: No best type found for implicitly-typed array
+                // 0.cs(5,26): error CS0029: Cannot implicitly convert type 'C' to 'Unknown?'
                 //         var a1 = new[] { x1, y1 };
-                Diagnostic(ErrorCode.ERR_ImplicitlyTypedArrayNoBestType, "new[] { x1, y1 }").WithLocation(5, 18),
-                // (7,18): error CS0826: No best type found for implicitly-typed array
+                Diagnostic(ErrorCode.ERR_NoImplicitConv, "x1").WithArguments("C", "Unknown?").WithLocation(5, 26),
+                // 0.cs(7,30): error CS0029: Cannot implicitly convert type 'C' to 'Unknown?'
```

Comment:
Not blocking this PR: should we also prevent this cascading diagnostic?

Reply:
I think that's a good idea. Opening #69138.